app/src/main/java/net/bible/android/view/activity/readingplan/actionbar/ReadingPlanTitle.kt

```diff
@@ -52,17 +51,18 @@ import javax.inject.Inject
 
 @ApplicationScope
 class ReadingPlanTitle @Inject
-constructor(private val readingPlanControl: ReadingPlanControl) {
+constructor() {
 
 
     private lateinit var actionBar: ActionBar
     private lateinit var activity: Activity
-        private set
+    fun isActivityInitialized() = ::activity.isInitialized
```

Comment:
I like property syntax more:

```suggestion
    val isActivityInitialized get() = ::activity.isInitialized
```

Reply:
though, why not use ::activity.isInitialized directly where needed?

Reply:
I needed it in DailyReading.kt, and for some reason Kotlin does not allow this type of code to be used from another class.
I've changed it to property. Let me know if there's a better way of doing it.

Reply:
Okay